examples/api/lib/widgets/shared_app_data/shared_app_data.0.dart

```diff
@@ -50,19 +53,27 @@ class _HomeState extends State<Home> {
               child: const Text('change foo'),
               onPressed: () {
                 _fooVersion += 1;
-                // Changing the SharedAppData's value for 'foo' causes the
-                // widgets that depend on 'foo' to be rebuilt.
+                // Changing the SharedAppData's value for 'foo' key causes the
+                // widgets that depend on 'foo' key to be rebuilt.
                 SharedAppData.setValue<String, String?>(
-                    context, 'foo', 'FOO $_fooVersion'); // no need to call setState()
+                  context,
+                  'foo',
+                  'FOO $_fooVersion',
+                ); // no need to call setState()
               },
             ),
             const SizedBox(height: 16),
             ElevatedButton(
               child: const Text('change bar'),
               onPressed: () {
                 _barVersion += 1;
+                // Changing the SharedAppData's value for 'bar' key causes the
+                // widgets that depend on 'bar' key to be rebuilt.
                 SharedAppData.setValue<String, String?>(
-                    context, 'bar', 'BAR $_barVersion'); // no need to call setState()
+                  context,
+                  'bar',
+                  'BAR $_barVersion',
+                ); // no need to call setState()
```

Comment:
Very minor: consider capitalizing the first letter and adding a final dot.
Same on line 62.

Reply:
@bleroux thanks for the review and the information. Have a great time!